assets/styl/_modules/_navbar.styl

```diff
@@ -24,22 +23,39 @@
         padding 0
 
         a
-            display block
             padding 0 1.8rem
             color $color-base
-            text-align center
+            display flex
             transition background-color .2s
```

Comment:
we should have a background-color because there is a transition on this property.

Reply:
right

Reply:
Transparent is black with 0 opacity or white with 0 opacity?
The transition is not the same.
It's a reason why when we use transition the property must be define in each state.
Second reason is units: transition from px to % or auto can be risky.